src/index.ts

```diff
@@ -1,3 +1,4 @@
+import { ExpressionStatement, Literal, Program } from 'estree'
```

Comment:
Can we have some test cases to check that the toggle works?

I have a hunch that it doesn't. Line 47 may want to check for `'Directive'`. https://github.com/estree/estree/blob/master/es5.md#directive

Reply:
The toggle works, though I agree it doesn't always work as expected, and there's a nuanced difference in how you would toggle it for interpreter errors and errors that result from breaking Source specs. I have yet to fix that. Meanwhile I'll start pulling over commits from my other branch which does have the test cases. I've also implemented the Directive check, though I had to implement my own interface since the ESTree index that we have doesn't seem to export any such interface.